docs/build/build-hrmp-channels.md

```diff
@@ -34,6 +35,11 @@ In order to dispatch a call from its sovereign origin, a parachain may use gover
 encoded call in a `Transact` instruction to the Relay Chain, but it may also execute this logic
 autonomously (e.g. on the notification that a channel was requested).
 
+The xcm-executor pallet can be forked by parachains' teams and modified to include logic to handle
+[high-level HRMP instructions](https://paritytech.github.io/xcm-docs/journey/channels-and-bridges.html#hrmpnewchannelopenrequest).
+These instructions are not included in the default version of the pallet, but can be used for
+opening, accepting, and closing channels without relying on governance.
```

Comment:
While this document gives concrete instructions on how to open HRMP channels between chains, this section falls short explaining what specific logic needs to be added to handle high-level HRMP instructions. Perhaps, a link to the forked code base and the successful opening of channels without relying on governance would help.

Reply:
This suggestion came from my comments. We're open-sourcing the code at Polimec next Monday and could include it here. However, the codebase is still in development.

The wording is very good @ filippoweb3. It should be noted that the initial opening would probably not be done by a notification from the relay, since someone needs to send the `hrmpInitOpenChannel` extrinsic to the relay first for this to happen.

At Polimec we expect the other parachain to first initiate the opening from whatever means they want (probably sudo), and from this notification start our logic to check if we should accept and initiate the other direction channel.

Reply:
@JuaniRios thanks, we will leave this open until we can link to the instructions you will opensource :)

Reply:
@filippoweb3 @DrW3RK Sorry for the delay. We're open sourcing sometime this week, prob tomorrow

Reply:
@JuaniRios any news?

Reply:
@filippoweb3 our legal team is now reviewing the licences. ETA around Wednesday next week

Reply:
Our codebase is now open source!
Here are some interesting links you might want to use:
- [polimec-node repo](https://github.com/Polimec/polimec-node)
- [hrmp instruction implementation in xcm-executor fork](https://github.com/Polimec/polimec-node/blob/da9d1ee0062ead7a62f815647813ada48e4c2250/pallets/xcm-executor/src/lib.rs#L890)
-[impl of the new xcm-executor config types](https://github.com/Polimec/polimec-node/blob/da9d1ee0062ead7a62f815647813ada48e4c2250/pallets/funding/src/lib.rs#L1398)

Let me know if you have any questions :)

Reply:
@DrW3RK I added those links above, lmk if its ok

Reply:
Looks good to me

Reply:
Thank you @JuaniRios for contributing!